stroeer/core/v1/article.proto

```diff
@@ -60,22 +76,3 @@ enum AssetType {
   ASSET_TYPE_URL = 3;
   ASSET_TYPE_METADATA = 4;
 }
-
-message Asset {
-  AssetType type = 1;
-  int32 index = 2;
-  map<string, string> fields = 4;
-  Metadata metadata = 5;
-}
-
-message Author {
-  string first_name = 1;
-  string surname = 2;
-  string web_url = 3;
-}
-
-message Onward {
-  string id = 1;
-  repeated string roles = 2;
```

Comment:
Hatte mit boba abgesprochen dass wir diesmal onward roles einführen, damit ‘onward’ allgemein ist, und die role aussagt ob zb editorial, more like this, etc.

Reply:
yes, but IMO we don't need that role in the `core.Article` since the CMS layer only knows about editorial onwards. We could add roles in the API of the aggregation layer.